apps/src/templates/account/AccountCard.tsx

```diff
@@ -24,32 +24,50 @@ const AccountCard: React.FunctionComponent<{
   buttonText: string;
   buttonType: ButtonType;
   href: string;
-}> = ({id, icon, title, content, buttonText, buttonType, href}) => (
+  iconList?: string[];
+}> = ({id, icon, title, content, buttonText, buttonType, href, iconList}) => (
   <Card data-testid={id}>
-    <CardHeader
-      title={title}
-      icon={
-        <FontAwesomeV6Icon
-          className={classNames(styles.cardIcon, 'fa-2x')}
-          iconName={icon}
-        />
-      }
-    />
-    <CardContent className={classNames(styles.cardContent)}>
-      {content}
-    </CardContent>
-    <CardActions>
-      <LinkButton
-        className={styles.button}
-        color={
-          buttonType === 'primary' ? buttonColors.purple : buttonColors.black
+    <div className={styles.contentWrapper}>
```

Comment:
I created this wrapper to allow for the card button(s) to stick to the bottom of the card to account for uneven content heights.